dolphinscheduler-api-test/pom.xml

```diff
@@ -37,7 +37,8 @@
 
         <junit.version>5.7.2</junit.version>
         <selenium.version>3.141.59</selenium.version>
-        <lombok.version>1.18.24</lombok.version>
+        <lombok.version>1.18.10</lombok.version>
+<!--        <lombok.version>1.18.20</lombok.version>-->
```

Comment:
What's the reason to downgrade this version?

Reply:
Thanks for the reminder, it has been restored

reson: 
Initially I modified the version of lombok because it did not work, but later found that it was caused by incorrect use, and has now been restored to the original version